app/serializers/concerns/v1/categorizable.rb

```diff
@@ -3,11 +3,7 @@ module V1::Categorizable
 
   class_methods do
     def get_categories(process)
-      if process.class == Workflow::Instance::Process && process.categories.empty?
-        process.definition&.categories&.map(&:name)
-      else
-        process.categories.map(&:name)
-      end
+      process.categories.map(&:name)
```

Comment:
Checked in production that all instances have their own categories copied over. No need to use the definition's.